themes/V3/Blank/snippets.js

```diff
@@ -128,6 +129,44 @@ module.exports = [
 			}
 		]
 	},
+
+	{
+		groupName : 'License',
```

Comment:
Rather than take up an entire snippet group for this which will probably not be used frequently, can I suggest we make these a sub-group under "Text Editor"?